src/components/MediaPlayer/VideoJS/components/js/VideoJSCurrentTime.js

```diff
@@ -47,12 +47,23 @@ function CurrentTimeDisplay({ player, options }) {
 
   const [currTime, setCurrTime] = React.useState(player.currentTime());
 
+  let initTimeRef = React.useRef(options.currentTime);
+  const setInitTime = (t) => {
+    initTimeRef.current = 0;
```

Comment:
Should this set to `t` instead of `0`?
```suggestion
    initTimeRef.current = t;
```